chromadb/utils/embedding_functions.py

```diff
@@ -144,22 +166,25 @@ def __init__(
                     api_key=api_key,
                     api_version=api_version,
                     azure_endpoint=api_base,
-                    default_headers=default_headers
+                    default_headers=default_headers,
                 ).embeddings
             else:
                 self._client = openai.OpenAI(
-                    api_key=api_key,
-                    base_url=api_base,
-                    default_headers=default_headers
+                    api_key=api_key, base_url=api_base, default_headers=default_headers
                 ).embeddings
         else:
             self._client = openai.Embedding
         self._model_name = model_name
         self._deployment_id = deployment_id
 
+    def max_input_length(self) -> int:
+        # Ref: https://platform.openai.com/docs/guides/embeddings/second-generation-models
+        return 8191
```

Comment:
is it possible that a user could still be using the first gen models?

Reply:
Entirely possible, but perhaps unlikely. We can altogether remove the OpenAI validations as their API already does token length checks and straight up rejects anything that's over the model's input seq.

Reply:
Ah ok